controllers/dspipeline_controller.go

```diff
@@ -225,8 +225,17 @@ func (r *DSPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	}
 
 	// Get Prereq Status (DB and ObjStore Ready)
-	dbAvailable, dbAvailableError := r.isDatabaseAccessible(ctx, dspa, params)
-	objStoreAvailable, objStoreAvailableError := r.isObjectStorageAccessible(ctx, dspa, params)
+	dbAvailableErrorMsg, objStoreAvailableErrorMsg := "", ""
+	dbAvailable, err := r.isDatabaseAccessible(ctx, dspa, params)
```

Comment:
rather than pass around `strings` for `errs` that already exist, I feel like we should update these to take in `_, dbAvailableErr`, and then have `GenerateStatus` to just take these errors in as parameters rather than a bool AND err that communicate the same thing.

Reply:
discussed in chat, we'll go with as is for now and follow up with a more cohesive pattern for passing this information around